test/module/irohad/consensus/yac/yac_mocks.hpp

```diff
@@ -139,9 +145,9 @@ namespace iroha {
 
         MockYacPeerOrderer() = default;
 
-        MockYacPeerOrderer(const MockYacPeerOrderer &rhs) {};
+        MockYacPeerOrderer(const MockYacPeerOrderer &rhs){};
```

Comment:
Is it changes happen with clang-format formatting? If no, check the whole pr and fix it.

Reply:
Yes, because of clang-formatting